docs/_documentation/platform/android/android-recyclerview.md

```diff
@@ -4,7 +4,7 @@ title: Android MvxRecyclerView
 category: Platforms
 ---
 
-Available in Android Support RecyclerView, MvvmCross 5.
+Available in Android Support RecyclerView, MvvmCross 4.
```

Comment:
In android-spinner.md it says "Available in MvvmCross 3." - should the numbering be consistent as here it says 4?

Reply:
RecyclerView was added later than the Spinner, why should it be consistent? This indicates in which version of MvvmCross this stuff is available.

Reply:
Your explanation make sense with regards to the different version numbers.

Reply:
👍